src/StoreApi/docs/order.md

```diff
@@ -0,0 +1,256 @@
+					"sizes": "(max-width: 800px) 100vw, 800px",
+					"name": "pennant-1.jpg",
+					"alt": ""
+				}
+			],
+			"variation": [],
+			"prices": {
+				"currency_code": "GBP",
+				"currency_symbol": "£",
+				"currency_minor_unit": 2,
+				"currency_decimal_separator": ".",
+				"currency_thousand_separator": ",",
+				"currency_prefix": "£",
+				"currency_suffix": "",
+				"price": "1105",
+				"regular_price": "1105",
+				"sale_price": "1105",
+				"price_range": null,
+				"raw_prices": {
+					"precision": 6,
+					"price": "11050000",
+					"regular_price": "11050000",
+					"sale_price": "11050000"
+				}
+			},
+			"item_data": [],
+			"totals": {
+				"currency_code": "GBP",
+				"currency_symbol": "£",
+				"currency_minor_unit": 2,
+				"currency_decimal_separator": ".",
+				"currency_thousand_separator": ",",
+				"currency_prefix": "£",
+				"currency_suffix": "",
+				"line_subtotal": "1105",
+				"line_subtotal_tax": "0",
+				"line_total": "884",
+				"line_total_tax": "0"
+			},
+			"catalog_visibility": "view"
+		}
+	],
+	"needs_payment": true,
+	"needs_shipping": true,
+	"totals": {
+		"subtotal":"2105",
+		"total_discount": "421",
+		"total_shipping": "500",
+		"total_fees": "0",
+		"total_tax": "0",
+		"total_refund": "0",
+		"total_price": "2184",
+		"total_items": "2105",
+		"total_items_tax": "0",
+		"total_fees_tax": "0",
+		"total_discount_tax": "0",
+		"total_shipping_tax": "0",
+		"tax_lines": []
+	},
+	"errors": [],
+	"payment_requirements": [ "products" ],
+}
+```
+
+### Error Response
+
+If an order action cannot be performed, an error response will be returned. This will include a reason code and an error message:
+
+```json
+{
+	"code": "woocommerce_rest_invalid_order",
+	"message": "Invalid order ID or key provided.",
+	"data": {
+		"status": 401
+	}
+}
+```
+
+## Get Order
+
+```http
+GET /order/{ORDER_ID}?key={KEY}&billing_email={BILLING_EMAIL}
+```
+
+There is one required parameter for this endpoint which is `key`. `billing_email` must be added for guest orders.
+
+```sh
+curl "https://example-store.com/wp-json/wc/store/v1/order/{ORDER_ID}?key={KEY}&billing_email={BILLING_EMAIL}"
+```
+
+Returns the full order object response (see [Order Response](#order-response)).
+
+<!-- FEEDBACK -->
+
+---
+
+[We're hiring!](https://woocommerce.com/careers/) Come work with us!
+
+🐞 Found a mistake, or have a suggestion? [Leave feedback about this document here.](https://github.com/woocommerce/woocommerce-blocks/issues/new?assignees=&labels=type%3A+documentation&template=--doc-feedback.md&title=Feedback%20on%20./src/StoreApi/docs/cart.md)
```

Comment:
```suggestion
🐞 Found a mistake, or have a suggestion? [Leave feedback about this document here.](https://github.com/woocommerce/woocommerce-blocks/issues/new?assignees=&labels=type%3A+documentation&template=--doc-feedback.md&title=Feedback%20on%20./src/StoreApi/docs/order.md)
```

The feedback link appears to link to a different doc. We should point it to this doc to avoid confusion.